official/resnet/resnet_run_loop.py

```diff
@@ -104,13 +103,13 @@ def process_record_dataset(dataset,
 
   # Defines a specific size thread pool for tf.data operations.
   if datasets_num_private_threads:
+    options = tf.data.Options()
```

Comment:
btw just FYI, this will fail right now if used with mirrored strategy with keras, because it expects batch etc to be the last dataset transformation. so you may observe a failure if `datasets_num_private_threads` is actually set

Reply:
@guptapriya   I think I can move this up and have the dataset.prefetch be last.  I will ask Jiri.